tests/rspack-test/__snapshots__/Config.test.js.snap

```diff
@@ -0,0 +1,81 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`config configCases/source-map/extract-source-map should pass 1`] = `
+Array [
+  webpack:///./external-source-map.txt,
+  webpack:///./extract2.js,
+]
+`;
+
+exports[`config configCases/source-map/extract-source-map should pass 2`] = `
+Array [
+  webpack:///./external-source-map.txt,
+  webpack:///./extract3.js,
+]
+`;
+
+exports[`config configCases/source-map/extract-source-map should pass 3`] = `
+Array [
+  webpack:///./external-source-map.txt,
+  webpack:///./extract3.js,
+]
+`;
+
+exports[`config configCases/source-map/extract-source-map-css should pass 1`] = `
+Array [
+  /*!*****************!*\\
+  !*** ./app.css ***!
+  \\*****************/
+* {
+	box-sizing: border-box; }
+  
+  .row {
+	display: flex;
+	margin-right: -15px;
+	margin-left: -15px; }
+  
+  .col-inner {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	color: #fff;
+	height: 50px;
+	background: coral; }
+  
+  .col-s3 {
+	flex: 0 1 25%;
+	padding: 0 15px; }
+  
+  
+  
+
+/*# sourceMappingURL=bundle0.css.map*/,
+  /*!*****************!*\\
```

Comment:
This is a little different with webpack.

Webpack only have one CSS, but we've got two.

See: https://github.com/webpack/webpack/pull/19824/files#diff-de61f5c1c819fec0a01a522d41e75824ba8e13f758920980349790b701ebe95cR11366